### PR TITLE
(For Aaron) PHP: Add handle for next_ref in getAccount

### PR DESCRIPTION
**Pull request by nickhurlburt**
@abeppu cc @yschatzberg

### CODE REVIEW

**Comment by abeppu** [comment]
As with the ruby and python libs, seems fine, but please update this test : https://github.com/SiftScience/sift-partner-php/blob/master/test/SiftClientTest.php#L175

**Comment by nickhurlburt** [comment]
test updated.

File: test/SiftClientTest.php [hunk 1]
```diff
@@ -173,7 +185,7 @@ private function validGetAccountsListResponseJson() {
                   )
                 ),
                 "has_more"=> False,
-                "total_results"=> 2,
```
Comment (abeppu) [inline test/SiftClientTest.php:176]: IRL the total results would still be 1 though, right?

**Comment by nickhurlburt** [comment]
Updated.

**Comment by abeppu** [comment]
LGTM

**Pull request merged**